src/components/landing-page/use-navigate-to-production.ts

```diff
@@ -9,9 +10,11 @@ export const useNavigateToProduction = (
   const navigate = useNavigate();
 
   useEffect(() => {
-    if (joinProductionOptions) {
+    if (joinProductionOptions && !isMobile) {
+      navigate("/production-calls");
+    } else if (joinProductionOptions && isMobile) {
       navigate(
-        `/production/${joinProductionOptions.productionId}/line/${joinProductionOptions.lineId}`
+        `/production-calls/production/${joinProductionOptions.productionId}/line/${joinProductionOptions.lineId}`
```

Comment:
Do we need several routes? Maybe we can have the isMobile check in the component?